public/app/features/provisioning/ConfigForm.tsx

```diff
@@ -206,6 +213,22 @@ export function ConfigForm({ data }: ConfigFormProps) {
       <Field label={'Linting'}>
         <Switch {...register('linting')} id={'linting'} />
       </Field>
+      <Field label={'Delete Policy'}>
```

Comment:
lets remove the frontend changes, and we can iterate on how this actually executes in the backend